.github/workflows/bundle-sync.yml

```diff
@@ -47,16 +44,39 @@ jobs:
           ref: ${{ github.ref_name }}  # Use the same branch name as in the event
           token: ${{ steps.app-token.outputs.token }}
 
+      - name: Generate bundle files
+        run: |
+          cd source_repo
+          ODH_PLATFORM_TYPE=rhoai make bundle
+
       - name: Sync Bundle Files
         run: |
           mkdir -p target_repo/to-be-processed/bundle
           echo "Updating bundle related files"
-          cp -r source_repo/bundle/* target_repo/to-be-processed/bundle
+          rm -rf target_repo/to-be-processed/bundle/*
+          if [ -d source_repo/rhoai-bundle ]; then
+            cp -r source_repo/rhoai-bundle/* target_repo/to-be-processed/bundle
+          else
+            cp -r source_repo/bundle/* target_repo/to-be-processed/bundle
+          fi
```

Comment:
Will this be ever used for non-rhoai bundle? We are calling `ODH_PLATFORM_TYPE=rhoai make bundle`, so I guess we count only with `rhoai-bundle`.